packages/core/@generated/graphql.ts

```diff
@@ -1560,6 +1578,7 @@ export type ClientSearchSuggestionsQueryQuery = {
           value: any
           valueReference: any
         }>
+        advertisement: { adId: string; adResponseId: string } | null
```

Comment:
**question**: Was this line updated manually?

Reply:
No wasn't. This line is based on this fragment [here](https://github.com/vtex/faststore/pull/2438/files#diff-695d985512e5faeafc13059164976eef7e894c1f06ab05066f8796e86c3313d3R203).